Graphics/Implicit/Primitives.hs

```diff
@@ -259,19 +262,23 @@ instance Object SymbolicObj2 ℝ2 where
     implicit a b= EmbedBoxedObj2 (a,b)
 
 instance Object SymbolicObj3 ℝ3 where
```

Comment:
one thing i was wondering about when looking at the code yesterday is: is there a good reason the Object class isn't exported?

Reply:
Not AFAIK. To hide `getImplicit` is my guess?

Reply:
PS we're hanging out in #ImplicitCAD on freenode. It's a better place to ask the maintainer things :)

Reply:
PPS: It is exported as of last week: https://github.com/colah/ImplicitCAD/blob/master/Graphics/Implicit.hs#L19. There just hasn't been a release since then!